action.yml

```diff
@@ -138,5 +145,7 @@ runs:
     '${{ inputs.tag_env_var_cmd }}', '${{ inputs.tag_env_var }}',
     '${{ inputs.tag_only_cmd }}', '${{ inputs.tag_only }}',
     '${{ inputs.use_latest_task_def_cmd }}', '${{ inputs.use_latest_task_def }}',
-    '${{ inputs.max_definitions_cmd }}', '${{ inputs.max_definitions }}'
+    '${{ inputs.max_definitions_cmd }}', '${{ inputs.max_definitions }}',
+    '${{ inputs.run_task_cmd }}', '${{ inputs.run_task }}',
+    '${{ inputs.wait_for_success_cmd }}', '${{ inputs.wait_for_success }}'
```

Comment:
On this in my initial commit (afe18b9) I've added the trailing comma, but then I noticed there wasn't one before my change, so to leave it as it was and not risk to introduce a breaking change I removed it. Do you happen to know if a trailing comma in that list would break things?

Reply:
That's a great question. I researched the validity of a trailing comma in yaml and I found that it is accepted. I didn't specifically look for Github actions.yml documentation on that, though. As far as I'm aware, we don't use ecs-deploy as a Github action, so I don't have an easy way to check that.

Reply:
Nice, for checking that maybe using a tool like https://github.com/nektos/act could help.